src/views/PaymentNetworkFees.vue

```diff
@@ -0,0 +1,202 @@
+<script setup lang="ts">
+import snapshot from '@snapshot-labs/snapshot.js';
+
+const {
+  DEFAULT_CURRENCY,
+  DEFAULT_PLAN,
+  CURRENCIES,
+  PLANS,
+  BASE_PRICE,
+  BASE_UNIT,
+  BASE_CURRENCY,
+  fxRates,
+  transfer,
+  paymentTx,
+  loading,
+  fxLoadStatus
+} = usePayment(import.meta.env.VITE_DEFAULT_NETWORK);
+const { web3Account } = useWeb3();
+const modalPostPaymentOpen = ref(false);
+
+const data = reactive({
+  tos: false,
+  contract_signed: false,
+  currency: DEFAULT_CURRENCY,
+  plan: DEFAULT_PLAN
+});
+
+const amount = computed(() => {
+  return computePrice(data.currency, PLANS[data.plan].unit);
+});
+
+function setData(key: string, value: string | boolean) {
+  data[key] = value;
+}
+
+function pay() {
+  transfer(amount.value, data.currency);
+}
+
+function computePrice(
+  currencyId: string | null,
+  unit: number,
+  discount = 0
+): number {
+  const fxRate = currencyId ? fxRates[currencyId] : 1;
+  let price = (BASE_PRICE / fxRate) * unit;
+
+  if (discount > 0) {
+    price = price - (price / 100) * discount;
+  }
+
+  return price;
+}
+
+function computePlanFiatPrice(plan, unit?: number): number {
+  return computePrice(null, unit || plan.unit, plan.discount);
+}
+
+function formatFiatCurrency(amount: number): string {
+  return `${BASE_CURRENCY.name} ${
+    BASE_CURRENCY.symbol
+  }${amount.toLocaleString()}`;
+}
+
+function formatCryptoCurrency(amount: number, currency): string {
+  return `~${currency.code} ${amount.toLocaleString()}`;
+}
+
+watch(paymentTx, () => {
+  modalPostPaymentOpen.value = !!paymentTx.value;
+});
+</script>
+
+<template>
+  <TheLayout class="max-w-lg px-4 md:px-0">
+    <BaseMessageBlock
+      v-if="!web3Account"
+      level="warning"
+      class="mb-3"
+      is-responsive
+    >
+      Connect your wallet first to proceed
+    </BaseMessageBlock>
+
+    <h1>Network fees</h1>
+    <div class="text-lg mb-4">Pay to support your network on Snapshot</div>
```

Comment:
```suggestion
    <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot.</p>
```
To match our design system for sub heading on figma